app/routes/contact/components/ContactForm.tsx

```diff
@@ -98,10 +97,23 @@ const ContactForm = (props: Props) => {
       />
 
       <p>
-        Du kan velge å sende meldingen med anonym avsender. De som mottar
-        meldingen vil ikke få vite hvem som har opprettet meldingen. De vil da
-        heller ikke ha mulighet til å svare på meldingen. Ønsker om innsyn kan
-        ikke sendes inn anonymt.
+        Ved anonym innsendelse vil de som mottar meldingen ikke få vite hvem som
+        har opprettet den, men de vil da heller ikke ha mulighet til å svare. Av
+        den grunn kan ikke ønsker om innsyn sendes inn anonymt.
+      </p>
+      <p>
+        For mer sensitive saker eller hvis du ønsker å varsle anonymt uten
```

Comment:
Isn't the entire point that it is possible to be anonymous WITH the option to reply?

Reply:
mm yes that makes a lot of sense. Thanks for proofreading 😬